### PR TITLE
removing old interface to write handle

### PR DESCRIPTION
**Pull request by abuts**
This PR should be condidered #1435 is merged.  It addresses ticket #1403
These change remove all instances of internal write handle and `get_new_handle` functions from code base so all filebacked operations are performed through external write handle connected with PageOp class.

### CODE REVIEW

**Review by cmarooney-stfc — APPROVED** [review]

File: _test/test_sym_op/test_cut_sym_cube.m [hunk 1]
```diff
@@ -24,7 +24,7 @@
             obj@TestCase(name)
         end
 
-        function test_cut_sym_identity_stripped(obj)
```
Comment (oerc0122) [inline _test/test_sym_op/test_cut_sym_cube.m:27]: Shouldn't be in this PR.

File: horace_core/symop/@Symop/Symop.m [hunk 2]
```diff
@@ -159,16 +159,8 @@ function disp(obj)
...
-                    curr_page = pix.data;
-                    for i = numel(obj):-1:1
-                        sel = ~obj(i).in_irreducible(curr_page(1:3, :), proj{:});
-                        curr_page(1:3, sel) = obj(i).transform_vec(curr_page(1:3, sel));
-                    end
-                    pix = pix.format_dump_data(curr_page);
-                end
-                pix = pix.finalise();
+                error('HORACE:Symop:not_implemented', ...
+                    'Transforming filebased pixels is not currently implemented');
```
Comment (oerc0122) [inline horace_core/symop/@Symop/Symop.m:163]:
```suggestion
                    'Transforming file-backed pixels is not currently implemented');
```

File: horace_core/algorithms/cut.m [hunk 3]
```diff
@@ -66,11 +66,13 @@
     % We expect either a .sqw or .dnd file, throw an error otherwise.
     ldr = sqw_formats_factory.instance().get_loader(source);
     % If we're cutting, don't ever want to load whole file
-    sqw_dnd_obj=obj_from_faccessor(ldr, n_object, sqw_only, dnd_only);
+    sqw_dnd_obj=obj_from_faccessor(ldr, n_object, sqw_only, dnd_only, ...
```
Comment (oerc0122) [inline horace_core/algorithms/cut.m:69]: Shouldn't be in this PR.

File: _test/test_sym_op/test_cut_sym_cube.m (hunk 1 above)
Comment (abuts) [reply _test/test_sym_op/test_cut_sym_cube.m:27]: I would disagree with this approach.

**Pull request merged**